tests/transformations/test_configuration_transform.py

```diff
@@ -0,0 +1,74 @@
+import numpy as np
+from ase.data import atomic_numbers
+
+from kliff.dataset import Configuration
+from kliff.transforms.configuration_transforms import ConfigurationTransform
+
+
+class GlobalCoulombMatrix(ConfigurationTransform):
```

Comment:
Any reason why creating a new class to test it -- can we use any existing one to test it?

Reply:
I was thinking of something like the LennardJones model test of Model class, to have a trivial implementation of a children class for checking any inconsistency. Although LennardJones is a member of `models` module, I wan not sure if Coulomb matrix will be of much use. Either I can move it in main code, or remove it all together, as anyway Descriptor and graph transforms are being tested.

Reply:
This is ConfigurationTransform -- how is LennardJones (which is a model) related?

Reply:
It is not related. I meant similar design principle. Like how LJ extended Model class for a simple system to test all the Parent class routines, I was thinking this globalCM class can be used to test routines of parent ConfigurationTransform class.

Reply:
I see. 

We can then probably leave the CM stuff in the test folder.